### PR TITLE
add details page for eventing trigger and subscription

### PR DESCRIPTION
**Pull request by invincibleJai**
**Fixes**: 
- https://issues.redhat.com/browse/ODC-4995
- https://issues.redhat.com/browse/ODC-4996

**Analysis / Root cause**: 
- user were not able to see associated filter, broker and subscriber for triggers 
- user were not able to see associated channel and subscriber for triggers 

**Solution Description**: 
Added details page for Trigger/Subscription to show info about filters, broker/channel and subscriber.

**Screen shots / Gifs for design review**: 

- Trigger

image

- Subscriptions

image

@openshift/team-devconsole-ux

**Unit test coverage report**: 

image

**Test setup:**

**Browser conformance**: 

- [ ] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

**Comment by bmignano** [comment]
Thanks @invincibleJai , looks good to me! I know we are discussing potentially surfacing event sources too but that may be a follow on :)

**Comment by karthikjeeyar** [comment]
@invincibleJai Just asking out of curiosity, is it possible to use i18n here for the strings or it will be added in your other PR?

**Comment by invincibleJai** [comment]
> @invincibleJai Just asking out of curiosity, is it possible to use i18n here for the strings or it will be added in your other PR?

yeah i think once initial PR for respective Packages are in then all subsequent PRs should handle localisation.

**Comment by karthikjeeyar** [comment]
The heading title `Filter` seems to be slightly bigger than other titles, we should customize the component or create new to fit our needs.

File: frontend/packages/knative-plugin/src/components/pub-sub/details/SubscriptionDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <div className="col-sm-6">
+          <ResourceSummary resource={subscription} />
+        </div>
+        <div className="col-sm-6">
+          {subscription.spec?.channel && (
+            <DynamicResourceLink
+              title="Channel"
+              name={subscription.spec.channel.name}
+              namespace={subscription.metadata.namespace}
+              kind={referenceForModel(EventingChannelModel)}
```
Comment (karthikjeeyar) [inline frontend/packages/knative-plugin/src/components/pub-sub/details/SubscriptionDetails.tsx:27]: This needs to be dynamic channel kind, use `getChannelRef(kind)`

**Comment by karthikjeeyar** [comment]
Verified locally, works fine
/lgtm

**Pull request merged**